File: Day 1 : Data Types/Solution.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() 
{
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";
    
    // Declare second integer, double, and String variables.
    int j;
    double e;
    string t;
  
    // Read and save an integer, double, and String to your variables.
    cin >> j;
    cin >> e;
    getline(cin >> ws, t); //ws ignores the whitespaces
    
    // Print the sum of both integer variables on a new line.
    cout << i + j << endl;
    
    // Print the sum of the double variables on a new line.
    cout << fixed << setprecision(1) << d + e << endl; 
  
    // Concatenate and print the String variables on a new line
    cout << s + t << endl;

    return 0;
}

File: Day 1 : Data Types/Solution.js
function main() {
    var i = 4
    var d = 4.0
    var s = "HackerRank "
  
    // Declare second integer, double, and String variables.
    var j = parseInt(readLine());
    var e = parseFloat(readLine());
    var t = readLine();

    // Print the sum of both integer variables on a new line.
    console.log(i + j);

    // Print the sum of the double variables on a new line.
    console.log((d + e).toFixed(1));
    
    // Concatenate and print the String variables on a new line
    console.log(s + t);
}

main();
